scss/_base/_forms.scss

```diff
@@ -156,19 +156,98 @@ select {
   }
 }
 
-// Checkboxes and radio boxes. Must be enclosed in `.option-group` class.
+// Checkboxes and radio boxes.
 //
 // Styleguide Forms - Option Fields
-.option-group, .form-item.form-type-checkbox {
-  margin-bottom: $base-spacing / 2;
 
-  input[type="checkbox"], input[type="radio"] {
-    display: inline;
+.option {
+  position: relative;
+  display: inline-block;
+  height: auto;
+  padding-left: $base-spacing;
+  margin: ($base-spacing / 4) 0;
+  cursor: pointer;
+
+  .option__indicator {
+    display: none;
   }
 
-  label {
-    display: inline;
-    padding-right: $base-spacing;
+  input {
+    position: absolute;
+    top: 2px;
+    left: 0;
   }
 }
 
+.checked.label-click .option {
```

Comment:
Might wanna go for that whole `.modernizr-checked` `.modernizr-label-click` class prefixing soon... I fear class collisions! :blue_car: :boom: :car: